api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSource.java

```diff
@@ -42,7 +43,16 @@
  * The default config sources always available by default are:
  * <ol>
  * <li>System properties (ordinal=400)</li>
- * <li>Environment properties (ordinal=300)</li>
+ * <li>Environment properties (ordinal=300)
+ *    <p>Depending on the operating system type, environment variables with '.' are not always allowed.
```

Comment:
please remove this as the change in mp config needs to be brought back to config jsr, global replacement not only .

Reply:
you might have not synced up with master